Makefile

```diff
@@ -165,9 +164,23 @@ check-all-generated-files-up-to-date: generate-all
 	@# generate-groups.sh can't find the lincense header when using Go modules, so we provide one
 	printf "/*\n%s\n*/\n" "$$(cat LICENSE)" > $@
 
-.PHONY: generate-ami
-generate-ami: ## Generate the list of AMIs for use with static resolver. Queries AWS.
-	time go generate ./pkg/ami
+### Update AMIs in ami static resolver
+.PHONY: update-ami
+update-ami: ## Generate the list of AMIs for use with static resolver. Queries AWS.
+	go generate ./pkg/ami
```

Comment:
Any reason why we're no longer timing this?

Reply:
Do you look at it? I find all those annoying and I'm not really looking at them. I can add it if we are going to use it

Reply:
I'm actually thinking about removing all of the `time` commands....

Reply:
>Do you look at it? 

Not really. I asked because the other ones are being timed.